sui-execution/latest/sui-move-natives/src/test_scenario.rs

```diff
@@ -229,6 +229,17 @@ pub fn end_transaction(
                     .or_default()
                     .insert(id);
             }
+            Owner::ConsensusV2 { authenticator, .. } => {
+                // TODO-DNS is this right to treat consensusv2 as address-owned for now in these tests?
```

Comment:
i see you didn't mean to submit this which i agree with

Reply:
removed